confignode/src/main/java/org/apache/iotdb/confignode/manager/ConfigManager.java

```diff
@@ -720,18 +757,68 @@ public TDataPartitionTableResp getOrCreateDataPartition(
     }
 
     DataPartitionResp queryResult =
-        partitionManager.getOrCreateDataPartition(getOrCreateDataPartitionReq);
-
+        partitionManager.getOrCreateDataPartition(getOrCreateDataPartitionPlan);
     resp = queryResult.convertToTDataPartitionTableResp();
 
-    LOGGER.debug(
-        "GetOrCreateDataPartition success. receive PartitionSlotsMap: {}, return: {}",
-        getOrCreateDataPartitionReq.getPartitionSlotsMap(),
-        resp);
+    if (CONF.isPartitionDebugEnabled()) {
+      printNewCreatedDataPartition(getOrCreateDataPartitionPlan, resp);
+    }
 
     return resp;
   }
 
+  private void printNewCreatedDataPartition(
+      GetOrCreateDataPartitionPlan getOrCreateDataPartitionPlan, TDataPartitionTableResp resp) {
+    StringBuilder partitionSlotsMapString = new StringBuilder("{");
+    for (Map.Entry<String, Map<TSeriesPartitionSlot, TTimeSlotList>> databaseEntry :
+        getOrCreateDataPartitionPlan.getPartitionSlotsMap().entrySet()) {
+      String database = databaseEntry.getKey();
+      partitionSlotsMapString.append("\n\tDatabase=").append(database).append(": {");
```

Comment:
use `%n` rather than `\n`. Please refer to [stackoverflow](https://stackoverflow.com/questions/31981136/difference-between-n-and-n-for-printing-a-new-line-in-java)

Reply:
The `%n` is only supported in String.format(). Not in StringBuilder.

Reply:
I've used System.lineSeparator() instead.